src/plugins/intel_gpu/src/kernel_selector/core/actual_kernels/reorder/reorder_kernel_base.cpp

```diff
@@ -152,10 +152,11 @@ ReorderKernelBase::DispatchData ReorderKernelBase::SetDefault(const reorder_para
     DispatchData dispatchData;
 
     auto& input = params.inputs[0];
+    auto& output = params.output;
     DataTensor input_tensor = input;
     // Image formats reorders use read_image and write_image functions that operate on 4 channels at once, and support only single batch,
     // make sure that reorder size is equal to spatials sizes only
-    if (params.inputs[0].GetLayout() == DataLayout::image_2d_rgba || params.output.GetLayout() == DataLayout::image_2d_rgba) {
+    if (input.GetLayout() == DataLayout::image_2d_rgba || output.GetLayout() == DataLayout::image_2d_rgba) {
```

Comment:
Just a notice:
what about creating a local variables for input/output layouts like this: auto& input_l = input.GetLayout() ?
(and the same below for output layout)
To me, it will improve readability and reduce amount of code.

Reply:
Thanks, Mikhail! Agree, I'll fix it later when dl-bencmark tests will be finished.